Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -618,10 +620,29 @@ open class MapView: UIView {
         cameraAnimatorsRunnerEnablable.isEnabled = true
 
         updateDisplayLinkPreferredFramesPerSecond()
+
+        // this will make sure that display link is only running on an active scene in foreground,
+        // preventing metal view drawing on background if the view is added to window not on foreground
+        displayLink.isPaused = window.prefersDisplayLinkPaused || UIApplication.shared.prefersDisplayLinkPaused
+
         displayLink.add(to: .current, forMode: .common)
     }
 }
 
+private extension UIApplication {
+    var prefersDisplayLinkPaused: Bool { UIApplication.shared.applicationState != .active }
+}
+
+private extension UIWindow {
+    var prefersDisplayLinkPaused: Bool {
+        if #available(iOS 13, *), windowScene?.activationState != .foregroundActive {
+            return true
+        }
+
+        return false
+    }
+}
```

Comment:
I usually have a concern about extensions to public classes (even private extensions). 
Can we please double-check that the host app can have its own public/private extensions with the same names without any issues? 
A few years ago it was an issue in some specific configurations that involved public protocols.